builtin/logical/pki/path_fetch_keys.go

```diff
@@ -144,6 +145,11 @@ func buildPathKey(b *backend, pattern string, displayAttrs *framework.DisplayAtt
 								Description: `Key Type`,
 								Required:    true,
 							},
+							"subject_key_id": {
+								Type:        framework.TypeString,
+								Description: `Subject key identifier`,
```

Comment:
Maybe something like?

```suggestion
								Description: `RFC 5280 Subject Key Identifier of the public counterpart`,
```